tests/test_api.py

```diff
@@ -272,6 +272,11 @@ class TestGradeProcessor(BaseTests):
     def test_export(self, course_grade_factory_mock):  # pylint: disable=unused-argument
         processor = api.GradeCSVProcessor(course_id=self.course_id)
         rows = list(processor.get_iterator())
+        assert any('student_key' in row for row in rows)
```

Comment:
Nit: explain what you're testing though comment/docstring or by creating descriptive standalone test (e.g. `test_student_key_in_export`)